### ISSUE TITLE
add support for Jupyter README.ipynb

### ISSUE THREAD

**Issue by monperrus** [isaacs/github#1425]
For open-science repos, it would be great to have a README which is a Jupyter notebook.

Jupyter ipynb files are rendered when opened directly:
https://github.com/monperrus/notebooks/blob/master/Readme.ipynb

But they are not rendered when opened as repo README: https://github.com/monperrus/notebooks

Dear Github team, could you render README.ipynb files as main README?

Thanks!

**Comment by monperrus** [comment]
For the record, just received the default Github answer:

> Thanks for writing in to suggest this. I've recorded this as a feature request for our team.
> I can't promise if or when we might start rending ipynb readme files but your feedback is in good hands!
> Please let me know if you have any questions about this or anything else.

**Comment by rikedyp** [comment]
Commenting here to show support as I would also really like this feature!

**Comment by fhgd** [comment]
+1 for this really nice feature!

**Comment by eau-de-cologne** [comment]
This feature would be highly appreciated.

**Comment by paulegradie** [comment]
Agreed. For me, either support  latex  in github markdown, or  render jupyter notebooks as readme.ipynb. That would be really great to  have.

**Comment by juancsherrera** [comment]
Commenting here to show support  as well

**Comment by 62random** [comment]
Agreed, wouldn't be that hard either (i guess), since the notebook preview feature already exists..

**Comment by mpaloni** [comment]
Would be nice :)

ps. Just googled "github make readme file out of ipynb"

**Comment by interrogator** [comment]
Yep, this seems like it shouldn't be too difficult to implement, since there is already a github ipynb renderer ... and would be great for those who share repositories with people who aren't quite so git(hub) literate! Would be a great feature for data science especially.

**Comment by estepankevich** [comment]
Would be nice to have this feature, what is the status of this item?

**Comment by tg1482** [comment]
Would make all our lives easier!

**Comment by jamesmatthewhamilton** [comment]
This would reduce redundancy. For many the README ends up being a copy-and-paste of a Jupyter Notebook segments. Additionally, Jupyter Notebook supports Latex.

**Comment by Robaina** [comment]
Showing support for this feature request!! Would really change thigns for many of us!

**Comment by hbaqueiro** [comment]
This would be a really great feature! Either this or support latex equations in markdown.

**Comment by ildoonet** [comment]
Add one up vote!

**Comment by marlonrcfranco** [comment]
I would love to have my jupyter notebook as main README. That should not be difficult to implement, as .ipynb files seems rendering anywhere else instead of the main page. Up voted for sure! 🤩

**Comment by amefonaffia** [comment]
Would really love to have this feature as well!

**Comment by asanmateu** [comment]
+1 more vote

**Comment by ShivamThukral** [comment]
+1 upvote

**Comment by darshan-barapatre** [comment]
+1

**Comment by Ben-Jamin-Griff** [comment]
+1

**Comment by LachlanGray** [comment]
+1 plz

**Comment by drscotthawley** [comment]
After I found out the MathJaX in my README.md wouldn't render properly, I thought, "Oh, since GitHub supports Jupyter notebooks, I'll just replace it with a README.ipynb".... and ended up here.  Surprised this isn't already a thing.

**Comment by quadrismegistus** [comment]
+1 another vote, this would be so useful!

**Comment by donno2048** [comment]
+1

**Comment by pflajszer** [comment]
+ 1 from me! It would make life much easier!

Supporting LaTeX in *.md would solve many problems too

**Comment by Gravifer** [comment]
Even if this cannot be done, at least support rendering KaTeX in markdown files.

**Comment by amyshenin-tech** [comment]
+1 for notebooks as README

**Comment by junwenleong** [comment]
It has been 1.5 years, really hoping for this feature. It would be especially useful for many DS professionals to avoid that ipynb-only repo that a viewer would have to make a pointless click on

**Comment by boba-and-beer** [comment]
Not only would DS professionals benefit but it would help reduce unnecessary github bloating of copy-paste in Jupyter notebooks (i.e. not having to save the images in the repository and instead be able to render immediately from Notebook and then having a separate notebook elsewhere) 

It would also make showing data visualisations so much more practical! Happy to be a beta tester for this feature if required :)

**Comment by davidnavecarvalho** [comment]
I wholeheartedly support this item to move up GitHub's priority list!

**Comment by tomdries** [comment]
This should become a feature, +1

**Comment by sandstromviktor** [comment]
just do it
